issues_to_socrata.py

```diff
@@ -73,7 +73,7 @@ def convert_timestamps(issues):
 
 
 def get_zenhub_metadata(workspace_id, token, repo_id, timeout=60):
-    url = f"https://api.zenhub.io/p2/workspaces/{workspace_id}/repositories/{repo_id}/board"
+    url = f"https://api.zenhub.com/p2/workspaces/{workspace_id}/repositories/{repo_id}/board"
```

Comment:
🙌  I definitely found a couple Gists with the .io endpoint yesterday so it existed at some point.

Reply:
would have been nice to get a heads up! https://github.com/ZenHubIO/API/commit/3a5405bc058e64e1c3b79f79e85e121db54012e9